hikari/impl/bot.py

```diff
@@ -213,6 +212,15 @@ class GatewayBot(traits.GatewayBotAware):
     ```
     """
 
+    shards: typing.Mapping[int, gateway_shard.GatewayShard]
+    """Mapping of shards in this application instance.
+
+    Each shard ID is mapped to the corresponding shard instance.
+
+    If the application has not started, it is acceptable to assume the
```

Comment:
Should this doc string not be being inherited from traits.ShardAware

Reply:
It doesnt seem to be, most probably since it becomes an attribute instead of a property

Reply:
Well that shouldn't be an issue if you don't move where its being type hinting it for no reason but also cannot reproduce that behaviour

Reply:
I am still able to reproduce it unfortunately: 
![image](https://user-images.githubusercontent.com/29100934/140497907-c118e91e-a2c3-4c94-afe1-75e043ed19a8.png)

I got absolutely no idea why this happens

Reply:
It shouldn't be an issue if its being type hinted in the `__init__` though since pdoc just doesn't introspect those

Reply:
Nope, even if I take the typehint out it still loses the docstring for some reason. I really havent been able to figure out